packages/jest-matchers/src/toThrowMatchers.js

```diff
@@ -81,6 +85,15 @@ const matchers: MatchersObject = {
   toThrowError: createMatcher('.toThrowError'),
 };
 
+const createNotAFunctionError = (matcherName, value, actual) => {
+  return new Error(
+    matcherHint(matcherName, 'function', getType(value)) +
+      '\n\n' +
+      'Expected the matcher to call a function, but instead ' +
```

Comment:
Should this be: "Received value must be a function"?

(Also that reminds me that we have an error in jest-matcher-utils that says "Actual value must be a number", that should be "Received value must be a number."